eng/doc/fips/UserGuide.md

```diff
@@ -473,21 +647,25 @@ func Prime(rand io.Reader, bits int) (p *big.Int, err error)
 
 func Prime(rand io.Reader, bits int) (p *big.Int, err error)

```

Comment:
Noticed this is a duplicate func signature rather than the func doc.

Reply:
Done!